api/src/api/views/XmlView.py

```diff
@@ -0,0 +1,41 @@
+# Copyright 2017 the Isard-vdi project authors:
+#      Josep Maria Viñolas Auquer
+#      Alberto Larraz Dalmases
+# License: AGPLv3
+
+#!flask/bin/python
+# coding=utf-8
+from api import app
+import logging as log
+import traceback
+
+from uuid import uuid4
+import time,json
+import sys,os
+from flask import request
+from ..libv2.apiv2_exc import *
+from ..libv2.quotas_exc import *
+
+#from ..libv2.telegram import tsend
+def tsend(txt):
+    None
+from ..libv2.carbon import Carbon
+carbon = Carbon()
+
+from ..libv2.quotas import Quotas
+quotas = Quotas()
+
+from ..libv2.api_xml import ApiXml
+xml=ApiXml()
```

Comment:
Is better use whitespaces arround `=`.
```suggestion
xml = ApiXml()
```